### PR TITLE
Cypress is defined as a peer dependency

### LINKED ISSUE #100: Cypress ^7.4.0 as dependency causes downloading multiple cypress version when used with Cypress in other version
When I want to use cypress-tags with e.g. Cypress 8, then `npm i` downloads Cypress 8 (as it's my dev dependency) and Cypress 7.4.0 (as it's a cypress-tags dependency). Of course, Cypress 7.4.0 is not used at all, and due to the size of the Cypress package downloading it takes a significant amount of time. Proposed solution is to include Cypress as a peer dependency, preferably in `any` version.
```json
      "peerDependencies": {
        "cypress": "*"
      }
```

### PR DESCRIPTION
**Pull request by TomaszG**
Fixes #100

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -41,6 +40,9 @@
     "tsify": "^5.0.2",
     "typescript": "^4.3.5"
   },
+  "peerDependencies": {
+    "cypress": "*"
```
Comment (annaet) [inline package.json:44]: Can you restrict the Cypress version to be `>= 7.0` as I believe there were some changes made between v6 and v7.

Comment (TomaszG) [reply package.json]: Done.

**Comment by TylorMayfield** [comment]
Would be excited to see this approved, causing extended build times for us because of duplicate cypress dependencies versions

**Comment by annaet** [comment]
@TomaszG Sorry it's taken me so long to get round to this!

Looks like we'll also need to install the peer dependency in the `main.yml` file so the build can run.

You should just be able to add this bit of code below where we install the deps currently:

```
      - name: Install Peer Dependencies
        run: npm install cypress --no-save
```

Then once the build is passing I'm happy to merge this and get a new release out.

**Comment by TomaszG** [comment]
@annaet done, please verify whether it works fine :) Feel free to amend this PR if needed.

**Comment by annaet** [comment]
@TomaszG This should all be available in v0.3.0 now. 

I've tested it and all seems to be working as expected. 

Thanks again for the contribution!

**Pull request merged**